react/features/notifications/components/native/Notification.js

```diff
@@ -66,12 +66,17 @@ class Notification extends AbstractNotification<Props> {
      * @private
      */
     _renderContent() {
-        const { maxLines = DEFAULT_MAX_LINES, t, title, titleArguments, titleKey } = this.props;
+        const { maxLines = DEFAULT_MAX_LINES, t, title, titleArguments, titleKey, raiseHandNotification } = this.props;
```

Comment:
THis is a generic notification system, it should have absolutely zero knowledge of specific notification types. Please fix this elsewhere.

Reply:
Could I use a more generic option? Instead of raiseHandNotification use something like concatText.
Because the notification now displays the description if there's one otherwise the title. But never both (on native)